users/tests.py

```diff
@@ -11,13 +11,18 @@
 from plio.cache import get_cache_key
 
 
-class OtpAuthTestCase(BaseTestCase):
+class OtpAuthTestCase(BaseTestCase):  
     """Tests the OTP functionality."""
 
     def setUp(self):
         super().setUp()
         # unset client credentials token so that the subsequent API calls goes as guest
         self.client.credentials()
+    def test_request_otp_mobile_not_passed(self):
```

Comment:
all functions within classes should be separate by a space. Currently, there is no space before this function.